services/token-reward-service/.env

```
@@ -24,27 +24,9 @@ DB_NAME=course_db
 # BLOCKCHAIN CONFIGURATION (Optional)
 # =====================================================================
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
-ACCOUNT_PRIVATE_KEY=0x4fcea8c0f36ac7fd8f1033cae252d941176a4d530913150da6e5239129c5372d
-CONTRACT_ADDRESS=0xA7548A695FfaBeD23BCcA90408A304DA05C25087
-ESCROW_CONTRACT_ADDRESS=0x4Fe6eEFB23dc9F285293805E18a68655E11299C3
-REWARD_DEPOSIT_ADDRESS=0xF7d5606003F9C39E5692eB916B772E7618D732B1
-REWARD_LISTENER_START_BLOCK=0
+ACCOUNT_PRIVATE_KEY=0x22821a351140957d1c69a151ac67d691c8f2671991d1479ac80bdfa5723bf291
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)